tests/torchtune/models/clip/test_clip_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
How should I unit test the encoder given that the test won't have access to the CLIP weights? Should I just build an encoder with random weights and check that a simple forward and backward pass doesn't throw any exceptions?

Reply:
Yes I would first verify against a reference implementation that you get forward pass parity, then you can just randomly initialize a small CLIP and use the output as the "golden value" for the test

Reply:
What Rafi said